bee-hive/pyproject.toml

```diff
@@ -27,3 +27,6 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.pytest.ini_options]
 addopts = "-v -s"
+
+[tool.poetry.scripts]
+"beeAI" = "cli.beeAI:runcommand"
```

Comment:
If we do what I suggest above then this would be:
```toml
"beeAI" = "cli.beeAI:run_cli"
```

Reply:
updated as requested